python/kfserving/kfserving/__init__.py

```diff
@@ -13,88 +13,45 @@
-from kfserving.models.v1beta1_predictor_extension_spec import V1beta1PredictorExtensionSpec
-from kfserving.models.v1beta1_predictor_spec import V1beta1PredictorSpec
-from kfserving.models.v1beta1_predictors_config import V1beta1PredictorsConfig
-from kfserving.models.v1beta1_sk_learn_spec import V1beta1SKLearnSpec
-from kfserving.models.v1beta1_tf_serving_spec import V1beta1TFServingSpec
-from kfserving.models.v1beta1_torch_serve_spec import V1beta1TorchServeSpec
-from kfserving.models.v1beta1_trained_model import V1beta1TrainedModel
-from kfserving.models.v1beta1_trained_model_list import V1beta1TrainedModelList
-from kfserving.models.v1beta1_trained_model_spec import V1beta1TrainedModelSpec
-from kfserving.models.v1beta1_trained_model_status import V1beta1TrainedModelStatus
-from kfserving.models.v1beta1_transformer_config import V1beta1TransformerConfig
-from kfserving.models.v1beta1_transformer_spec import V1beta1TransformerSpec
-from kfserving.models.v1beta1_transformers_config import V1beta1TransformersConfig
-from kfserving.models.v1beta1_triton_spec import V1beta1TritonSpec
-from kfserving.models.v1beta1_xg_boost_spec import V1beta1XGBoostSpec
+# import models into kfserving package
+from .models.knative_addressable import KnativeAddressable
+from .models.knative_condition import KnativeCondition
+from .models.knative_url import KnativeURL
+from .models.knative_volatile_time import KnativeVolatileTime
+from .models.net_url_userinfo import NetUrlUserinfo
+from .models.v1alpha2_alibi_explainer_spec import V1alpha2AlibiExplainerSpec
+from .models.v1alpha2_aix_explainer_spec import V1alpha2AIXExplainerSpec
+from .models.v1alpha2_batcher import V1alpha2Batcher
+from .models.v1alpha2_custom_spec import V1alpha2CustomSpec
+from .models.v1alpha2_inference_service import V1alpha2InferenceService
+from .models.v1alpha2_inference_service_list import V1alpha2InferenceServiceList
+from .models.v1alpha2_inference_service_spec import V1alpha2InferenceServiceSpec
+from .models.v1alpha2_inference_service_status import V1alpha2InferenceServiceStatus
+from .models.v1alpha2_endpoint_spec import V1alpha2EndpointSpec
+from .models.v1alpha2_predictor_spec import V1alpha2PredictorSpec
+from .models.v1alpha2_transformer_spec import V1alpha2TransformerSpec
+from .models.v1alpha2_explainer_spec import V1alpha2ExplainerSpec
+from .models.v1alpha2_py_torch_spec import V1alpha2PyTorchSpec
+from .models.v1alpha2_sk_learn_spec import V1alpha2SKLearnSpec
+from .models.v1alpha2_pmml_spec import V1alpha2PMMLSpec
+from .models.v1alpha2_logger import V1alpha2Logger
+from .models.v1alpha2_onnx_spec import V1alpha2ONNXSpec
+from .models.v1alpha2_status_configuration_spec import V1alpha2StatusConfigurationSpec
+from .models.v1alpha2_triton_spec import V1alpha2TritonSpec
+from .models.v1alpha2_tensorflow_spec import V1alpha2TensorflowSpec
+from .models.v1alpha2_xg_boost_spec import V1alpha2XGBoostSpec
```

Comment:
we still need the v1beta1 objects
@jinchihe can you help review this also

Reply:
Is my problem, I am regenerating.

Reply:
Hi, I followed the documentation through the generate swagger.json through script `./hack/update-openapigen.sh`, but defintion is `"github.com.kubeflow.kfserving.pkg.apis.serving.v1beta1.AIXExplainerSpec`, how can I solved this problem.